core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.stories.ts

```diff
@@ -68,9 +68,21 @@ export default {
     ]
 };
 
-export const Default = () => ({
-    component: DotDeviceSelectorSeoComponent,
-    props: {
-        selected: action('selected')
-    }
-});
+export const Default = () => {
+    const handleClick = () => {
+        // open selector logic
+    };
+
+    return {
+        template: `
+        <button (click)="op.openMenu($event)">Open Selector</button>
+        <dot-device-selector-seo #op></dot-device-selector-seo>
```

Comment:
We have talk about opening the storybook to the public. I'm not really sure about the scope of that because we just talked about the idea, but would be awesome if you can use PrimeNG buttons in this template just in case.